ui/desktop/app/styles/app.scss

```diff
@@ -135,6 +135,32 @@
     margin: sizing.rems(xs);
   }
 
+  .button-window-close {
+    &:not(:disabled) {
+      &:hover {
+      background: var(--failure);
+      }
+    }
+  }
+
+  .button-window-minimize,
+  .button-window-fullscreen {
+    &:not(:disabled) {
+      &:hover {
+        background: var(--white);
+        color: var(--ui-gray-starker-4);
+
+        @media (prefers-color-scheme: dark) {
```

Comment:
We had a bug in the desktop client where hovering over the minimize/fullscreen btns did not trigger a color change. See video.

https://github.com/user-attachments/assets/ff737650-4786-496f-b5b5-fa8ae2a67b0b

Reply:
Good catch but a few questions: 
* What scenario would we need the `not(:disabled)`?
* We shouldn't need another media query for dark mode, the tokens should have automatically handled it
  *  Since it's already always a black toolbar/buttons, do we need these to be changing colors when switching between light/dark?

Reply:
- Fair question. Those btns do not need a setting to account for being disabled.
- Correct, light/dark-mode specific settings were overcomplicating the fix. Our `var(--white)` and `var(--black)` are not overridden in the application, so I used those instead. Tested this at the system and electron app level and the fix works.